terracumber_config/tf_files/SUSEManager-4.2-build-validation.tf

```diff
@@ -1224,21 +1224,14 @@ module "sles11sp3-terminal" {
   providers = {
     libvirt = libvirt.terminus
   }
-  source             = "./modules/minion"
+  source             = "./modules/pxe_boot"
   base_configuration = module.base_retail.configuration
-  product_version    = "4.2-released"
   name               = "terminal-sles11sp3"
   image              = "sles11sp4" # This is not a typo
```

Comment:
I know it says it's not a typo but do you know the reason why we're using sles11sp4 for a sles11sp3 terminal?

Reply:
You build the sle 11 sp 3 terminal on a sle 11 sp 4 build host, because there is no such thing as a sle 11 sp 3 build host.

(I hope the explanation is correct - for further insights, ask Vladimir)